src/schema/ecommerce/types/order.ts

```diff
@@ -139,14 +139,6 @@ const orderFields = {
     type: GraphQLString,
     description: "Buyer phone number",
   },
-  lastOffer: {
-    type: OfferType,
-    description: "Latest offer",
-  },
-  offers: {
-    type: OfferConnection,
-    description: "List of submitted offers made on this order so far",
-  },
   offerTotalCents: {
```

Comment:
Not related to this PR but this is another field we can deprecate, not sure about the usage in other systems but i think it's not used anymore

Reply:
Good eye! 

I'll do a check of client usage again - if it's not used anywhere, I'll remove that field in this PR.

If it is used, and we need to clean those up first...we'll talk about it in slack.

Reply:
awesome! thanks! sorry i should have founded that one earlier

Reply:
I don't see it used anywhere. In fact, it's been [removed from Exchange already](https://github.com/artsy/exchange/commit/d9670c7), so it wouldn't be working if anyone was using it 😸 

I'll remove it and include it in this PR, and update JIRA to reflect the addition of that field.

Reply:
I didn’t catch this at first, but it looks like there is an `offerTotal` in addition to `offerTotalCents`. I’m not seeing any usage of that one, either - can it be removed?

Reply:
I removed `offerTotal` and `offerTotalCents`. Let me know if I got it wrong!

Reply:
awesome! this makes sense!  `OfferTotalCents` is the field coming directly from Exchange, `OfferTotal` is the added field in MP which gets some arguments like `symbol` and etc. and returns the display string of that amount in cents. we should be safe, we've already switched it to use `itemsTotalCents` before https://github.com/artsy/metaphysics/blob/master/src/schema/ecommerce/types/order.ts#L154